jhub_apps/service/routes.py

```diff
@@ -174,6 +174,12 @@ async def start_server(
             username=user.name,
             servername=server_name,
         )
+        if response.status_code in [403, 404]:
+            raise HTTPException(
+                detail=f"User doesn't have permissions to start server: '{server_name}'",
```

Comment:
Should we add "or the server with this name does not exist"? I think having one status for both is good, just wanted to confirm if this works the way I think it does.